zeus/optimizer/batch_size/server/database/schema.py

```diff
@@ -0,0 +1,191 @@
+    mab_prior_precision: Mapped[float] = mapped_column(Float, default=0.0)
+    mab_num_exploration: Mapped[int] = mapped_column(Integer, default=2)
+    mab_seed: Mapped[Optional[int]] = mapped_column(Integer, nullable=True)
+
+    mab_random_generator_state: Mapped[Optional[str]] = mapped_column(
+        VARCHAR(length=300), nullable=True
+    )
+    exp_default_batch_size: Mapped[int] = mapped_column(Integer, nullable=False)
+
+    stage: Mapped[Stage] = mapped_column(Enum(Stage), default=Stage.Pruning)
+    min_cost: Mapped[Optional[float]] = mapped_column(Float, nullable=True)
+    min_batch_size: Mapped[int] = mapped_column(Integer, nullable=False)
+
+    batch_sizes: Mapped[list["BatchSize"]] = relationship(
+        order_by="BatchSize.batch_size.asc()",
+        back_populates="job",
+        # always fetch batch size(int) whenever we fetch the job.
+        # https://docs.sqlalchemy.org/en/14/orm/loading_relationships.html#relationship-loading-techniques
+        lazy="joined",
+    )
+
+
+class BatchSize(Base):
+    """Batch size states table schema. Represents one batch size of a job.
+
+    (job_id, batch_size) as a pk, and have three states(exploration, measurement, GaussianTs arm state) as fk.
+    For explorations and measurements, one-to-many relationship. For arm_state, one-to-(zero or one) relationship.
+    """
+
+    __tablename__ = "BatchSize"
+
+    job_id: Mapped[UUID] = mapped_column(ForeignKey("Job.job_id"), primary_key=True)
+    batch_size: Mapped[int] = mapped_column(Integer, primary_key=True)
+
+    explorations: Mapped[list["ExplorationState"]] = relationship(
+        back_populates="batch_size_state",
+        order_by="ExplorationState.round_number.asc()",
+    )
+    measurements: Mapped[list["Measurement"]] = relationship(
+        back_populates="batch_size_state", order_by="Measurement.timestamp.asc()"
+    )
+
+    arm_state: Mapped[Optional["GaussianTsArmState"]] = relationship(
+        back_populates="batch_size_state",  # populates GaussianTsArmState->BatchSize
+        # https://stackoverflow.com/questions/398697
+        # 93/when-do-i-need-to-use-sqlalchemy-back-populates#:~:text=didn%27t%20provide%20a-,back_populates,-argument%2C%20modifying%20one
+    )
+
+    job: Mapped["Job"] = relationship(back_populates="batch_sizes")
+
+
+class State(enum.Enum):
+    """Exploration state.
+
+    Exploring means we issued this batch size and waiting for the report.
+    Converged/Unconverged means the result of training with that batch size.
+    """
+
+    Exploring = "Exploring"
+    Converged = "Converged"
+    Unconverged = "Unconverged"
+
+
+class ExplorationState(Base):
+    """Exploration state table schema with (job_id, batch_size, round_number) as a pk. Represents an exploration state of a batch size.
+
+    Refer to `ExplorationStateModel`[zeus.optimizer.batch_size.server.batch_size_state.models.ExplorationStateModel] for attributes.
```

Comment:
```suggestion
    Refer to [`ExplorationStateModel`][zeus.optimizer.batch_size.server.batch_size_state.models.ExplorationStateModel] for attributes.
```
Ditto for a couple docstrings below.